samTemplate.yaml

```diff
@@ -7,7 +7,7 @@ Resources:
     Properties:
       Handler: lambda_function.lambda_handler
       Runtime: python3.6
-      CodeUri: alexa/
+      CodeUri: alexa/build/
```

Comment:
If the path has changed for the code should the code move to this directory or the build will fail?

Reply:
Annoyingly I didn't clock this until I'd sent the request - it's a change which is necessary for the clean building using 'make' with SAM local. The background for this use is in the URL below, with Heitor's comment especially useful:

https://github.com/awslabs/aws-sam-cli/issues/53#issuecomment-345307381

Given that it still has a few wrinkles and I hadn't finished testing it, I would keep the path without build/ for now (so changes on that line and line 25) so that it works as intended.